applications/GeoMechanicsApplication/tests/test_submerged_construction_of_excavation.py

```diff
@@ -0,0 +1,205 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.GeoMechanicsApplication.geomechanics_analysis as analysis
+import KratosMultiphysics.GeoMechanicsApplication.context_managers as context_managers
+from KratosMultiphysics.GeoMechanicsApplication.gid_output_file_reader import (
+    GiDOutputFileReader,
+)
+import test_helper
+
+import os
+
+
+class KratosGeoMechanicsSubmergedConstructionOfExcavation(KratosUnittest.TestCase):
+    stage_info = {
+        "initial_stage": {"time": -1.0, "name": "1_Initial_stage"},
+        "null_step": {"time": 0.0, "name": "2_Null_step"},
+        "wall_installation": {"time": 1.0, "name": "3_Wall_installation"},
+        "first_excavation": {"time": 2.0, "name": "4_First_excavation"},
+        "strut_installation": {"time": 3.0, "name": "5_Strut_installation"},
+        "second_excavation": {"time": 4.0, "name": "6_Second_excavation"},
+        "third_excavation": {"time": 5.0, "name": "7_Third_excavation"},
```

Comment:
Perhaps we can make the keys slightly clearer:
```suggestion
        "initial_stage": {"end_time": -1.0, "base_name": "1_Initial_stage"},
        "null_step": {"end_time": 0.0, "base_name": "2_Null_step"},
        "wall_installation": {"end_time": 1.0, "base_name": "3_Wall_installation"},
        "first_excavation": {"end_time": 2.0, "base_name": "4_First_excavation"},
        "strut_installation": {"end_time": 3.0, "base_name": "5_Strut_installation"},
        "second_excavation": {"end_time": 4.0, "base_name": "6_Second_excavation"},
        "third_excavation": {"end_time": 5.0, "base_name": "7_Third_excavation"},
```

Reply:
changed to end_time and base_name